### ISSUE TITLE
Add option to disable Google Translate in footer

### ISSUE THREAD

**Issue by trubach** [ATLAS-Illinois/Illinois-Framework-Theme#1065]
The Google Translate footer element has thrown some performance and a11y issues. There should be an option to disable it in the theme settings if a site admin would like to. By default, it will stay enabled.
